.vscodeignore

```diff
@@ -6,6 +6,7 @@ out/__tests__/**
 out/src/__mocks__/**
```

Comment:
Perhaps it would make sense to structure the `.vscodeignore` file to ignore everything, and only whitelist what is needed (example [here](https://github.com/Microsoft/vscode-vsce/issues/12#issuecomment-285964473)).

This could prevent new folders/files that are added to the repo in the future from accidentally getting bundled in the VSIX, but no need to change this now 😄

Reply:
@tjohnsonBCM This is an interesting suggestion and a valid point. I would rather address it as a separate issue, since it doesn't strictly have to do with Webpack. If others on the Zowe Explorer squad like the idea, I'd be happy to go along with it!

Reply:
@tjohnsonBCM Issue #883 filed to discuss/address this.